testdata/recreate/correctness.cql

```diff
@@ -0,0 +1,34 @@
+
+CREATE KEYSPACE gocqlx_correctness WITH replication = {
```

Comment:
Looks super simple, what does it test?

Reply:
If queries returned by `Recreate` are correct in terms of CQL language for all entities, and they're always the same after multiple executions. I can merge it with each entity test.